x-pack/plugins/uptime/public/components/monitor_management/action_bar/action_bar_portal_node.tsx

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createPortalNode } from 'react-reverse-portal';
+
+export const ActionBarPortalNode = createPortalNode();
```

Comment:
This seems duplicate to `uptime/public/pages/monitor_management/action_bar_portal_node.tsx`. 

I suggest however, this one should be used than the one defined under `/pages`. As the portal node is local to the UI components (to ActionBarPortal component). Plus, page components import UI components, whereas UI components shouldn't import stuff from page components.

Reply:
I misspoke. Using the version here shouldn't cause circular references, but as I mentioned it is used by both routes and `monitor_management/action_bar`, so I think it's good to keep it in pages.